maven-3.0.x/src/main/java/org/apache/maven/shared/transfer/repository/internal/Maven302RepositoryManager.java

```diff
@@ -20,18 +20,24 @@
  */
 
 import org.sonatype.aether.RepositorySystem;
-import org.sonatype.aether.RepositorySystemSession;
 import org.sonatype.aether.repository.LocalRepository;
 
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+
 /**
- * 
+ *
  */
-class Maven302RepositoryManager
-    extends Maven30RepositoryManager
+@Singleton
+@Named( "maven-3.0.2" )
```

Comment:
I think this can be dropped because we won't get lower than 3.0.5.

Reply:
fixed, as maven 3.0.x went to maven 3.0.5 (but kept original code, just make other abstract and this one MAVEN_3_0_X)